---
layout: default
title: Publications
subtitle: Publications in Wu Lab
permalink: /publication
---

# Publications

**You** can visit [Google Scholar](http://scholar.google.com/citations?user=pQF7BaYAAAAJ&hl=en) for more details<br>
\* Co-first authors          \# Co-corresponding authors

## Preprint

29\. **Jiang W-J**\*, Hu C\*, Yi X*, Xu Q, Lou T, **Wang H**, **Zhou J**, **Zhu H**, Li Y, Wen Y, Fan R, Shen J, Wong C.C.L\#, Zhen X\#, **Wu H-J**\# (2022). Assessing base-resolution DNA mechanics on the genome scale.[(HTML)](https://www.biorxiv.org/content/10.1101/2022.02.16.480786v1) 

## 2023

28\.  Landshammer A\*, Bolondi A\*, Kretzmer H, Much C, Buschow R, Rose A, **Wu H-J**, Mackowiak SD, Braendl B, Giesselmann P, Tornisiello R, Parsi KM, Huey J, Mielke T, Meierhofer D, Maehr R, Hnisz D, Michor F, Rinn JL, Meissner A\# (2023) T-REX17 is a transiently expressed non-coding RNA essential for human endoderm formation. eLife 12, e83077.[(PDF)](../_data/PAPER/29.pdf)

27\.  Cao Y\*, **Wang H**\*, Jin Z\*, Hang J, Jiang H\#, **Wu H-J**\#, Zhang Z\#. Characterization of Non-Obstructive Azoospermia in Men Using Gut Microbial Profiling. Journal of Clinical Medicine. 2023;12(2):701. [(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/blob/master/_data/PAPER/28.pdf)

## 2022

26\.  Wei N\*, Nie Y, Liu L\#, Zheng X\#, **Wu H-J**\#. Secuer: Ultrafast, scalable and accurate clustering of single-cell RNA-seq data.PLOS Computational Biology. 2022;18(12):e1010753. [(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/27.pdf)

25\. **Wu H-J**\*, Temko D\*, Maliga Z\*, Moreira A, Sei E, Minussi DC, Dean J, Lee C, Xu Q, Hochart G, Jacobson C, Yapp C, Schapiro D, Sorger P, Seeley EH, Navin N, Downey RJ, Michor F. Spatial intra-tumor heterogeneity is associated with survival of lung adenocarcinoma patients. Cell Genomics.2022;2(8):100165.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/26.pdf)

## 2021

24\. **Wu H-J**\*, Landshammer A/*, Stamenova EK, Bolondi A, Kretzmer H, Meissner A, et al. Topological isolation of developmental regulators in mammalian genomes. Nature Communications. 2021;12(1):4897.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/25.pdf)

## 2020
23\. Shu S\*, **Wu H-J**\*, Ge JY, Zeid R, Harris IS, Jovanović B, et al. Synthetic Lethal and Resistance Interactions with BET Bromodomain Inhibitors in Triple-Negative Breast Cancer. Molecular Cell. 2020;78(6):1096-113.e8.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/24.pdf)

## 2019
22\. Zhang W, Li Z, Wei N, **Wu H-J**, Zheng X. Detection of differentially methylated CpG sites between tumor samples with uneven tumor purities. Bioinformatics. 2019;36(7):2017-24.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/23.pdf)

## 2018
21\. Hinohara K\*, **Wu H-J**\*, Vigneau S, McDonald TO, Igarashi KJ, Yamamoto KN, et al. KDM5 Histone Demethylase Activity Links Cellular Transcriptomic Heterogeneity to Therapeutic Resistance. Cancer Cell. 2018;34(6):939-53.e9. [(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/22.pdf)

20\. Jun HJ\*, Appleman VA\*, **Wu H-J**\*, Rose CM, Pineda JJ, Yeo AT, et al. A PDGFRα-driven mouse model of glioblastoma reveals a stathmin1-mediated mechanism of sensitivity to vinblastine. Nature Communications. 2018;9(1):3116.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/21.pdf)

19\. Cimino PJ, Kim Y, **Wu H-J**, Alexander J, Wirsching H-G, Szulzewsky F, et al. Increased HOXA5 expression provides a selective advantage for gain of whole chromosome 7 in IDH wild-type glioblastoma. Genes & development. 2018;32(7-8):512-23.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/20.pdf)

## 2017
18\. Han L\*, **Wu H-J**\*, Zhu H\*, Kim K-Y, Marjani SL, Riester M, et al. Bisulfite-independent analysis of CpG island methylation enables genome-scale stratification of single cells. Nucleic Acids Research. 2017;45(10):e77-e.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/19.pdf)

17\. Riester M\*, **Wu H-J**\*, Zehir A, Gönen M, Moreira AL, Downey RJ, et al. Distance in cancer gene expression from stem cells predicts patient survival. PLOS ONE. 2017;12(3):e0173589.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/18.pdf)

16\. Zheng X, Zhang N, **Wu H-J**, Wu H. Estimating and accounting for tumor purity in the analysis of DNA methylation data from cancer studies. Genome Biology. 2017;18(1):17[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/17.pdf)

## 2016
15\. **Wu H-J**\*, Michor F. A computational strategy to adjust for copy number in tumor Hi-C data. Bioinformatics. 2016;32(24):3695-701.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/16.pdf)

14\. Mahajan N, **Wu H-J**, Bennett RL, Troche C, Licht JD, Weber JD, et al. Sabotaging of the oxidative stress response by an oncogenic noncoding RNA. FASEB J. 2017;31(2):482-90.[(PDF)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5240659/)

## 2015
13\. Zhang N\*, **Wu H-J**\*, Zhang W, Wang J, Wu H, Zheng X. Predicting tumor purity from methylation microarray data. Bioinformatics. 2015;31(21):3401-5.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/14.pdf)

## 2014
12\. Zheng X\*, Zhao Q\*, **Wu H-J**\*, Li W, Wang H, Meyer CA, et al. MethylPurify: tumor purity deconvolution and differential methylation detection from single tumor DNA methylomes. Genome Biology. 2014;15(7):419.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/13.pdf)

## 2013
<!-- 12\. **Hua-Jun Wu**\*, Zhi-Min Wang*, Meng Wang and Xiu-Jie Wang. Widespread Long Noncoding RNAs as Endogenous Target Mimics for MicroRNAs in Plants. Plant Physiology. 2013 Apr; vol. 161 no. 4 1875-1884.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/12.pdf) -->

11\. Fu S, Lv Z, Gao Z, **Wu H-J**, Pang J, Zhang B, et al. De novo centromere formation on a chromosome fragment in maize. 2013;110(15):6033-6.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/11.pdf)

10\. Zhang B, Lv Z, Pang J, Liu Y, …, **Wu H-J**, et al. Formation of a Functional Maize Centromere after Loss of Centromeric Sequences and Gain of Ectopic Sequences  The Plant Cell. 2013;25(6):1979-89.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/10.pdf)

9\. **Wu H-J**\*, Wang Z-M\*, Wang M, Wang X-J. Widespread Long Noncoding RNAs as Endogenous Target Mimics for MicroRNAs in Plants  Plant Physiology. 2013;161(4):1875-84[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/9.pdf)

8\. Ling H-Q, Zhao S, Liu D, Wang J, …, **Wu H-J**, …, Wang J, et al. Draft genome of the wheat A-genome progenitor Triticum urartu. Nature. 2013;496(7443):87-90[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/8.pdf)

## 2012
7\. Li W, Shuai L, Wan H, Dong M, …, **Wu H-J**, Liu Z-H, et al.Androgenetic haploid embryonic stem cells produce live transgenic mice. Nature. 2012;490(7420):407-11.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/7.pdf)

6\. **Wu H-J**\*, Ma Y-K\*, Chen T, Wang M, Wang X-J. PsRobot: a web-based plant small RNA meta-analysis toolbox. Nucleic Acids Research. 2012;40(W1):W22-W8.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/6.pdf)

5\. **Wu H-J**\*, Zhang Z\*, Wang J-Y\*, Oh D-H\*, Dassanayake M\*, Liu B\*, et al. Insights into salt tolerance from the genome of Thellungiella salsuginea. 2012;109(30):12219-24.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/5.pdf)

4\. Sheng C, Zheng Q, Wu J, …,**Wu H-J**, …,Wang X-J, et al. Direct reprogramming of Sertoli cells into multipotent neural stem cells by defined factors. Cell Research. 2012;22(1):208-18.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/4.pdf)

## 2011
3\. Tong M, Lv Z, Liu L, Zhu H, …, **Wu H-J**, …,Wang X-J, et al. Mice generated from tetraploid complementation competent iPS cells show similar developmental features as those from ES cells but are prone to tumorigenesis. Cell Research. 2011;21(11):1634-7.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/3.pdf)

## 2010
2\. Liu L, Luo G-Z, Yang W, …, **Wu H-J**,Wang Liu, et al. Activation of the Imprinted Dlk1-Dio3Region Correlates with Pluripotency Levels of Mouse Stem Cells. Journal of Biological Chemistry. 2010;285(25):19483-90.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/2.pdf)

## 2009
1\. Wuriyanghan H, Zhang B, …,**Wu H-J**, …,Wang X-J,et al.  The Ethylene Receptor ETR2 Delays Floral Transition and Affects Starch Accumulation in Rice. The Plant Cell. 2009;21(5):1473-94.[(PDF)](https://github.com/multiomics-bjmu/multiomics-bjmu.github.io/raw/master/_data/PAPER/1.pdf)

